python/pyspark/streaming/tests.py

```diff
@@ -1636,7 +1636,13 @@ def search_kinesis_asl_assembly_jar():
         jars = "%s,%s,%s,%s,%s" % (kafka_assembly_jar, flume_assembly_jar, mqtt_assembly_jar,
                                    mqtt_test_jar, kinesis_asl_assembly_jar)
 
-    os.environ["PYSPARK_SUBMIT_ARGS"] = "--jars %s pyspark-shell" % jars
+    # We need to set userClassPathFirst here because the streaming data source classes are also
```

Comment:
Doesn't this mean that `SPARK_PREPEND_CLASSES` won't work for these classes, though?